### PR TITLE
Adds Kleibkhar Map

### PR DESCRIPTION
**Pull request by NataKilar**
Adds a new testing map on a habitable planet for easier testing

### CODE REVIEW

File: maps/kleibkhar/kleibkhar.dm [hunk 1]
```diff
@@ -0,0 +1,24 @@
+#if !defined(USING_MAP_DATUM)
+	#ifdef UNIT_TEST
+		#include "unit_tests.dmm"
+	#endif
```
Comment (Andrew-Fall) [inline maps/kleibkhar/kleibkhar.dm]: Don't need this anymore

File: maps/kleibkhar/unit_tests.dmm [hunk 2]
```diff
@@ -0,0 +1,19 @@
+"a" = (/turf/space,/area/space)
```
Comment (Andrew-Fall) [inline maps/kleibkhar/unit_tests.dmm]: This map can be removed

**Review by Andrew-Fall — CHANGES_REQUESTED** [review]

File: maps/kleibkhar/kleibkhar_exoplanet.dm [hunk 3]
```diff
@@ -0,0 +1,108 @@
+#define EXT_EDGE_GRASS_KLEIBKHAR    (53 * 0.001)
```
Comment (Andrew-Fall) [inline maps/kleibkhar/kleibkhar_exoplanet.dm]: Shouldn't this be EXT_LAYER_CONSTANT or something?

Comment (NataKilar) [reply maps/kleibkhar/kleibkhar_exoplanet.dm]: This follows from the other layer defines, but its more explicit in those files I guess. I'll go ahead and change it.

Comment (Andrew-Fall) [reply maps/kleibkhar/kleibkhar_exoplanet.dm]: I meant (53 * EXT_LAYER_CONSTANT)

Comment (NataKilar) [reply maps/kleibkhar/kleibkhar_exoplanet.dm]: EXT_LAYER_CONSTANT is undefined in an earlier file, so I can't use it here. Lmk if there's anything else blocking and I'll add a comment or something.

**Review by Andrew-Fall — APPROVED** [review]

**Pull request merged**